### ISSUE TITLE
Misassignment of B.1 for B.1.617.2 sample

### ISSUE THREAD

**Issue by embatty** [cov-lineages/pango-designation#106]
Pangolin using default pangoLEARN calls sample EPI_ISL_2433611 as B.1. The scorpio call is B.1.617.2-like with 12 ALT alleles, and USHER mode also calls B.1.617.2. The closely related sample EPI_ISL_2433599 is called as B.1.617.2 by all methods - the single SNP difference between them is A10285G, so I think that must be causing pangoLEARN to assign it to B.1.

```
taxon,lineage,conflict,ambiguity_score,scorpio_call,scorpio_support,scorpio_conflict,version,pangolin_version,pangoLEARN_version,pango_version,status,note
hCoV-19/Thailand/DMSc-00431/2021|EPI_ISL_2433611|2021-05-15,B.1,0.0,1.0,Delta (B.1.617.2-like),1.000000,0.000000,PLEARN-v1.2.12,3.0.5,2021-06-05,v1.2.12,passed_qc,scorpio call: Alt alleles 12; Ref alleles 0; Amb alleles 0
```

**Comment by Jamesm1118** [comment]
I'm having the same issue with a sample that is calling B.1 with Scorpio calling B.1.617.2-like with 12 ALT alleles. Nothing unusual with AA substitutions/deletions - all have been present before in other B.1.617.2 samples. The only SNP that doesn't seem to be familiar is C4582T.

**Comment by rmcolq** [comment]
The default mode is sensitive to ambiguous bases and new diversity. If scorpio makes the call B.1.617.2 you can assume it is that. Glad to hear that the new `--usher` mode worked for you in this context - this is precisely why we have added it! If your mis-assigned sequences have few ambiguous bases and are on GISAID we can add them to the training set to improve future calls

**Comment by rmcolq** [comment]
This is resolved since pangolin v3.1 - pangolin lineage now represents the scorpio call unless it is a child of it so in this case you should get a lineage assignment of B.1.617.2
